tensorflow/lite/kernels/CMakeLists.txt

```diff
@@ -252,8 +251,6 @@ set(TEST_WITH_EXTERNAL_MAIN_LIST
   pow_test.cc
   quant_basic_lstm_test.cc
   quantize_test.cc
-  random_standard_normal_test.cc
```

Comment:
Non-existing kernel tests were removed.

Reply:
It's resolved by https://github.com/tensorflow/tensorflow/pull/54486